broker/README.md

```diff
@@ -0,0 +1,46 @@
+# Mosquitto MQTT Broker
+
+## Running the broker
+
+Start by building the docker image
+
+```
+docker build -t flotilla-broker .
+```
+
+The broker may be started as
+
+```
+docker run -it -p 1883:1883 -p 9001:9001 flotilla-broker:latest
+```
+
```

Comment:
Maybe also add an example of how to test the docker image, by subscribing and publishing to a topic?

Reply:
```suggestion

docker run -it -p 1883:1883 -p 9001:9001 flotilla-broker:latest

mosquitto_sub -t 'topic' -u mosquitto -P default
mosquitto_pub -t 'topic' -m 'hello world' -u mosquitto -P default
```

Reply:
Done.